openeogeotrellis/job_registry.py

```diff
@@ -755,68 +770,73 @@ def create_job(
                 description=description,
             )
         if self.elastic_job_registry:
-            with self._just_log_errors("create_job", job_id=job_id):
-                ejr_job_info = self.elastic_job_registry.create_job(
-                    process=process,
-                    user_id=user_id,
-                    job_id=job_id,
-                    title=title,
-                    description=description,
-                    api_version=api_version,
-                    job_options=job_options,
-                )
+            ejr_job_info = self.elastic_job_registry.create_job(
+                process=process,
+                user_id=user_id,
+                job_id=job_id,
+                title=title,
+                description=description,
+                api_version=api_version,
+                job_options=job_options,
+            )
         if zk_job_info is None and ejr_job_info is None:
             raise DoubleJobRegistryException(f"None of ZK/EJR registered {job_id=}")
         return zk_job_info or ejr_job_info
 
     def get_job(self, job_id: str, user_id: str) -> dict:
         # TODO: eliminate get_job/get_job_metadata duplication?
-        ejr_job = None
-        if self.elastic_job_registry:
-            with self._just_log_errors("get_job", job_id=job_id):
-                with contextlib.suppress(JobNotFoundException):
-                    ejr_job = self.elastic_job_registry.get_job(job_id=job_id)
-
-        if ejr_job:
-            return ejr_job
-
+        zk_job = ejr_job = None
         if self.zk_job_registry:
```

Comment:
I haven't changed the order, it's just about being able to turn off `ZkJobRegistry` on CDSE by means of a config flag.

The changes on these particular lines are just because I rolled back the quickfix from earlier this week (which actually reversed the order but in a non-configurable way).